analysis/import/UnderstandImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/understand/UnderstandProjectBuilder.kt

```diff
@@ -108,15 +248,9 @@ class UnderstandProjectBuilder(
     private fun ProjectBuilder.insertRow(rawRow: Array<String?>, header: UnderstandCSVHeader) {
         try {
             val row = UnderstandCSVRow(rawRow, header, pathSeparator)
-            if (row.isFileRow) {
-                this.insertByPath(row.pathInTree(), row.asNode())
-            }
+            this.insertByPath(row.pathInTree(), row.asNode())
         } catch (e: IllegalArgumentException) {
-            logger.warn { "Ignoring row due to ${e.message}" }
+            logger.warn { "Ignoring $rowNumber-th row ${Arrays.toString(rawRow)} due to: ${e.message}" }
```

Comment:
When running the build I get 3106 WARN messages like the one below about ignoring rows. Do these show up on purpose? 
```
[main] WARN de.maibornwolff.codecharta.importer.understand.UnderstandProjectBuilder - Ignoring 3106-th row [Function, selected.(unnamed_function_9), visualization/app/codeCharta/ui/legendPanel/legendPanelController.spec.js, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, 1, null, 1, null, 0, null, 0, null, 1, 1, null, null, null, 0, null, 0, null, null, 1, 0, null, null, 0, 0, null, 1, 1, null, 1, 1, 1, 1, null, null, null, null, null, null, null, 0, null, null, null, null, null, null, null] due to: Kind Function not supported, yet.
```

Reply:
I is on intension. Maybe a different log level (Info) would be ok, as it is an anticipated issue with a workaround.

Reply:
yea I think another log level is a good idea